tools/clang/test/hash_stablity_script/hash_stability.test

```diff
@@ -0,0 +1 @@
+RUN: python3 %S/../../../../utils/hct/HashStability.py -a %S/../DXC/Inputs/smake.hlsl -p %llvm_tools_dir
```

Comment:
A replacement for the hash stability script would be less gross. Also we could put this under `tools/clang/test/tools/HashStability` which at least kinda matches the patterns we have in LLVM's test directories.